test/acceptance/features/steps/steps.py

```diff
@@ -425,3 +426,17 @@ def etc_cluster_is_running(context, etcd_name):
         print("etcd cluster not present, creating etcd cluster")
         assert etcd_cluster.create() is True, "etcd cluster is not created"
         assert etcd_cluster.is_present() is True, "etcd cluster is not present"
+
+
+@when(u'Invalid Service Binding is applied')
+def invalid_sbr_is_applied(context):
+    yaml = context.text
+    sbr = ServiceBinding()
+    output = sbr.attempt_to_create(yaml)
+    context.error = output
```

Comment:
Please rename the context variable to `context.expected_error` so that it is clear that it is expected instead of being confused with an ordinary error.

Reply:
Done.